static/openapi/vwrs-spec.json

```diff
@@ -1463,10 +983,10 @@
       "get": {
         "summary": "Get API key by unique key_id",
         "tags": [
-          "API Key"
+          "API key"
         ],
         "description": "Gets API key by unique key_id.",
-        "operationId": "getAPIKey",
+        "operationId": "getAPI Key",
```

Comment:
can operationIDs have spaces in them?

Reply:
No. I fixed create and update, but somehow missed this.